.github/pull_request_template.md

```diff
@@ -4,6 +4,6 @@
 
 ## How was this change tested? 🤨
 
-⚡ ⚠ If this change has cross service impact, ***run [integration test create_preassembly_image_spec.rb](https://github.com/sul-dlss/infrastructure-integration-test) on stage as it tests preservation*** and/or test in stage environment, in addition to specs. ⚡
+⚡ ⚠ If this change has cross service impact, or if it changes [code used in the replication pipeline](https://github.com/sul-dlss/preservation_catalog/blob/main/app/jobs/README.md), ***run [integration test create_preassembly_image_spec.rb](https://github.com/sul-dlss/infrastructure-integration-test) on stage as it tests preservation*** and/or test in stage environment, in addition to specs. ⚡
```

Comment:
refer to "jobs" and other spots in the code directly, in addition to link?

Also, would like one more blank line after -- my bad for only having 2.   If you want to leave this PR as is, okay to leave it at 2.

Reply:
ok, i added another blank line, and i fiddled with the text a bit.  i tried to list the most relevant classes, mostly jobs, with a link to the replication overview that encourages a glance if the author is unsure.

initially i kept the examples of what's replication-related in a parenthetical before the testing advice, but it felt like it read a little better as a follow-on explanatory sentence.  other version is in this orphan commit:  https://github.com/sul-dlss/preservation_catalog/blob/5a0a89d25f4f15c2a19bf6dc6fb948acd30e90e2/.github/pull_request_template.md

what this PR has:
<img width="1016" alt="Screen Shot 2022-02-25 at 9 25 10 PM" src="https://user-images.githubusercontent.com/7741604/155830462-77719a53-e560-4b75-833b-6e671cdae5c2.png">


the version i decided not to use:
<img width="1017" alt="Screen Shot 2022-02-25 at 9 25 30 PM" src="https://user-images.githubusercontent.com/7741604/155830465-6c72c462-5f3a-4be3-bf63-431ccb2de545.png">


tested the links and they work.

merging since you approved and hopefully this addresses the feedback, but wouldn't mind a second look to confirm.